cibuildwheel/macos.py

```diff
@@ -47,31 +47,41 @@ def call(args, env=None, cwd=None, shell=False):
 
         # if this version of python isn't installed, get it from python.org and install
         python_package_identifier = 'org.python.Python.PythonFramework-%s' % config.version
+        installation_bin_path = '/Library/Frameworks/Python.framework/Versions/{}/bin'.format(config.version)
         if python_package_identifier not in installed_system_packages:
             # download the pkg
             call(['curl', '-L', '-o', '/tmp/Python.pkg', config.url])
             # install
             call(['sudo', 'installer', '-pkg', '/tmp/Python.pkg', '-target', '/'])
 
+        # Python bin folders on Mac don't symlink python3 to python, so we do that
+        # so `python` and `pip` always point to the active configuration.
+        if os.path.exists('/tmp/cibw_bin'):
+            shutil.rmtree('/tmp/cibw_bin')
+        os.makedirs('/tmp/cibw_bin')
+
+        if config.version[0] == '3':
```

Comment:
If we do this in our own directory anyway, we could even just do this for Python 2 as well (i.e., `'python' + config.version[0]`). But I've no clue if that's better or worse, and more robust against further changes or not. So more of a thought/consideration than of a review remark.